concent_api/core/views.py

```diff
@@ -70,6 +91,11 @@ def send(_request, client_message):
                 golem_message,
                 message_timestamp,
             )
+            store_message_auth(
+                golem_message,
+                provider_public_key  = force_task_to_compute.last().auth.provider_public_key,
+                requestor_public_key = client_public_key,
+            )
```

Comment:
Why would it fail if `force_task_to_compute` query ever returns more than one object ?

Reply:
It would not fail and that's the point. We want it to fail if it ever happens. It's a `filter()` so if there are two objects in the database it will return two.

Assertions aren't there for perfect code. They're there to catch bugs. And such a situation would be a bug that we want to detect.

Reply:
Changed to use Message.objects.get and proper exceptions handling.

Reply:
You don't need `get()` here because we want to first check if the object exists. An assertion would be enough. Please use them more.

Reply:
:+1: